lib/node_modules/@stdlib/math/stats/anova1/lib/mean_table.js

```diff
@@ -0,0 +1,71 @@
+'use strict';
+
+// MODULES //
+
+var incrstdev = require( '@stdlib/math/stats/incr/stdev' );
+var uniqueElems = require('./unique_elems.js');
+
+/**
+* Computes the vector of means for each treatment.
+*
+* @param {NumericArray} x - Measured values
+* @param {Array} factor - Array of treatments
+* @returns {{}} destination array
+*/
+function meanTable(x, factor) {
+	var treatments;
+	var factorCount;
+	var meanTable;
```

Comment:
Can we rename the `meanTable` variable? As-is, it has the same name as the function, which can be confusing.

Reply:
`tableOfMeans`

Reply:
You probably don't have to be as descriptive. It would be enough to name the variable `out`. The meaning of the variable is readily available from the function docs, and, as it stands now, one has to read until function end to know what value is returned from the function.